server/views/partials/recommendation/recallTypeFTR.njk

```diff
@@ -4,12 +4,51 @@
     For sentences under 12 months, recalls must be fixed term unless either:
     <ul>
         <li>the person has been charged with a serious offence (murder or an offence listed in
-            <a target="_blank" rel="noopener noreferrer" href="https://www.legislation.gov.uk/ukpga/2020/17/schedule/18">Schedule 18 of the Sentencing Act 2020</a>)
+            <a target="_blank" rel="noopener noreferrer" href="https://www.legislation.gov.uk/ukpga/2020/17/schedule/18">Schedule 18
+                of the Sentencing Act 2020</a>)
         </li>
         <li>the MAPPA level is 2 or 3 at the point of recall</li>
     </ul>
     This applies to people aged 18 and over only.
 </p>
+<p>
+    You have said that:
+    <ul>
+        <li>{{ recommendation.personOnProbation.name }}
+            <strong>is</strong>
+            {% if recommendation.isUnder18 == false or recommendation.isUnder18 == 'NO' %}
```

Comment:
Have we not got this typed?  Surely it comes from the API/Database as boolean.

Reply:
isUnder18?: boolean,
  isMappaLevelAbove1?: boolean,
  isSentence12MonthsOrOver?: boolean,
  hasBeenConvictedOfSeriousOffence?: boolean,

Reply:
corrected - I did double check after @DarrenW-HMPPS 's comment. Thanks @robinandrewdevilliers .